tabcmd/commands/datasources_and_workbooks/datasources_and_workbooks_command.py

```diff
@@ -42,3 +42,76 @@ def get_wb_by_content_url(logger, server, workbook_content_url) -> TSC.WorkbookI
         if len(matching_workbooks) < 1:
             Errors.exit_with_error(logger, message=_("dataalerts.failure.error.workbookNotFound"))
         return matching_workbooks[0]
+
+    @staticmethod
+    def apply_values_from_url_params(request_options: TSC.PDFRequestOptions, url, logger) -> None:
```

Comment:
it is a string, yes. It's not a full url, it's the representation of the workbook/view as it would be in a url (tabcmd get "/views/Sales_Analysis/Sales_Report.png") - I'm actually not quite sure how we handle encoding yet and have to do some comparison with behaviors in the original tabcmd, but it's possible a proper library will be better.